### ISSUE TITLE
🐛[BUG][NPC/HNM] King Behemoth - Kick Out concerns

### ISSUE THREAD

**Issue by AuburnRDM** [HorizonFFXI/HorizonXI-Issues#134]
Kick Out seems to not be working properly in accordance to [FFXIclopedia](https://ffxiclopedia.fandom.com/wiki/King_Behemoth) information:
- The range of the attack seemed to be no more than about 10' (As opposed to the listed 40')
- The rear cone of the attack is also extremely narrow--standing right behind the feet did not make me get hit by the attack while the target was centered behind the boss. I can't recall if this is meant to be the case, but it felt off.

I'm not sure if the distance is an intended change, incorrect wiki information, or a bug meant to be fixed.

**Additional context**
This was tested with a group of multiple players while feeding the boss TP.

**Comment by hookstar** [comment]
Indeed, even clopedia has contradicting information. The main KB page lists 
[Kick Out](https://ffxiclopedia.fandom.com/wiki/Kick_Out) --- Used when hate is generated from behind King Behemoth. 40' range. Affects everyone in range. Causes [Blind](https://ffxiclopedia.fandom.com/wiki/Blind_(Status_Effect)).

While the kickout page has the below listed.


Utsusemi/Blink absorb: 2-3 shadows
Range: Unknown cone, backwards
Notes: Only used when the [Behemoth](https://ffxiclopedia.fandom.com/wiki/Category:Behemoths) is attacking with it's tail.

Does anyone have a current retail with 2x chars to maybe pop KB and have a secondary character take increments of 10 yalms backwards? I do believe the current meta is to fight from behind to ONLY have it use kick out?
